### DISCUSSION TITLE
Removing Column With 'View Discussion' Button

### DISCUSSION THREAD

**Discussion by POAcom** [requarks/wiki#3805] [General]
Hi

Does anyone know how to remove the 'column' highlighted in orange in the attached image from screen?

* Don't want the view discussion, last edited etc items there
* May want to keep the share and print icons but not in this column

Just want to remove what is in orange box in the attached image from screen.

Is it even possible?

Thanks,

[Doc13.pdf](https://github.com/Requarks/wiki/files/6275087/Doc13.pdf)

**Comment by Rainshaw** [comment]
Maybe you can make your own theme...

**Comment by Max2983** [comment]
I have the same problem. I need to hide the 'TALK' box from the left.
Also need to move 'Last edited..' and bookmark/share/printer icons block up above or down below the article text.
Can help anyone?
